apps/mobile/src/components/loading/fetch-wrapper.tsx

```diff
@@ -0,0 +1,28 @@
+import { Error, FetchState, Loading } from '@/components/loading';
+
+import { assertUnreachable } from '@leather.io/utils';
+
+interface FetchWrapperProps {
```

Comment:
Better as `unknown` to express we don't know or really care what this is?

Reply:
Updating this in [LEA-2627](https://linear.app/leather-io/issue/LEA-2627/refactor-balance-component-and-show-locked-balances)